### PR TITLE
Port :maps.from_keys/2 to JS

### LINKED ISSUE #364: Port :maps.from_keys/2 to JS

### PR DESCRIPTION
**Pull request by Sorc96**
Closes #364

### CODE REVIEW

**Comment by bartblast** [comment]
I've just added "Quality Checks Before Submitting" section here: https://hologram.page/reference/contributing, though to get the updated checks you'll need to sync with the `dev` branch.

File: assets/js/erlang/maps.mjs [hunk 1]
```diff
@@ -33,6 +33,19 @@ const Erlang_Maps = {
   // End fold/3
   // Deps: []
 
+  // Start from_keys/2
+  "from_keys/2": (keys, value) => {
+    if (!Type.isList(keys)) {
```
Comment (bartblast) [inline assets/js/erlang/maps.mjs:38]: This needs to be a proper list, see `Type.isProperList()`

File: test/elixir/hologram/ex_js_consistency/erlang/maps_test.exs [hunk 2]
```diff
@@ -47,6 +47,18 @@ defmodule Hologram.ExJsConsistency.Erlang.MapsTest do
     end
   end
 
+  describe "from_keys/2" do
```
Comment (bartblast) [inline test/elixir/hologram/ex_js_consistency/erlang/maps_test.exs:50]: Can you add test cases for boundary conditions:
* empty list of keys
* single key
* multiple keys

File: test/elixir/hologram/ex_js_consistency/erlang/maps_test.exs [hunk 3]
```diff
@@ -47,6 +47,18 @@ defmodule Hologram.ExJsConsistency.Erlang.MapsTest do
...
+    test "creates a map with the given keys and value" do
+      assert :maps.from_keys([:a, :b], 1) === %{a: 1, b: 1}
+    end
+
+    test "raises ArgumentError if the first argument is not a list" do
```
Comment (bartblast) [inline test/elixir/hologram/ex_js_consistency/erlang/maps_test.exs:63]: We'll need 2 tests for type validation:
* not a list
* improper list

File: test/javascript/erlang/maps_test.mjs [hunk 4]
```diff
@@ -121,6 +121,31 @@ describe("Erlang_Maps", () => {
     });
   });
 
+  describe("from_keys/2", () => {
```
Comment (bartblast) [inline test/javascript/erlang/maps_test.mjs:124]: Comments for Elixir consistency tests apply here too.

**Review by bartblast — CHANGES_REQUESTED** [review]

**Comment by bartblast** [comment]
Thank you @Sorc96! :)
